### PR TITLE
Add ALSO IoT Platform

### PR DESCRIPTION
**Pull request by epiller**
#### Summary

Added ALSO IoT Platform.

#### Changes

- Added ALSO IoT Platform webhook template.

### CODE REVIEW

File: also-iot-platform.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+template-id: also-iot-platform
+name: ALSO IoT Platform
+description: A commercial-ready IoT solution to deploy or resell to your customers immediately
+logo-url: https://also.allthingstalk.com/assets/img/brand/also/logo-full-2-372baab0a0.svg 
+info-url: https://www.allthingstalk.com/iot-also-resellers
+documentation-url: https://docs.allthingstalk.com/networks/use-the-things-network-v3/
+fields:
+  - id: network_token
+    name: Network token
+    description: Network token from your device's connectivity settings.
```
Comment (nejraselimovic) [inline also-iot-platform.yml:10]:
```suggestion
    description: Network token from your device's connectivity settings on ALSO IoT
```

File: also-iot-platform.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+template-id: also-iot-platform
```
Comment (nejraselimovic) [inline also-iot-platform.yml:1]: You need to add this template ID to templates.yml file

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

File: also-iot-platform.yml (hunk 1 above)
Comment (epiller) [reply also-iot-platform.yml]: Done

**Comment by epiller** [comment]
Requested changes done.

**Comment by nejraselimovic** [comment]
hey @epiller I'm so sorry but I somehow managed to mess up this PR :') I replaced it with #88. While waiting for the approval to merge it, do you happen to have any documentation for this?

**Comment by epiller** [comment]
Opusteno 😂
Our main documentation is here: https://docs.allthingstalk.com and it applies to all of our instances (AllThingsTalk Maker, ALSO IoT Platform, others).

**Comment by epiller** [comment]
@nejraselimovic I still don't see ALSO IoT Platform on TheThingsNetwork webhooks. Do you have any idea when it will be available? Thanks!

**Comment by nejraselimovic** [comment]
> @nejraselimovic I still don't see ALSO IoT Platform on TheThingsNetwork webhooks. Do you have any idea when it will be available? Thanks!

Hey Vanja, it should become available with the next TTS release. I see that the merge deadline for v3.28.0 is October 18, I guess it was postponed a bit because of the conference.

**Pull request closed without merging**